src/uu/uniq/src/uniq.rs

```diff
@@ -390,12 +411,24 @@ pub fn uu_app() -> Command {
                 .help("end lines with 0 byte, not newline")
                 .action(ArgAction::SetTrue),
         )
+        .group(
+            ArgGroup::new("obsolete_skip_field")
+            .multiple(false)
+            .args(["0","1","2","3","4","5","6","7","8","9"])
+        )
         .arg(
             Arg::new(ARG_FILES)
                 .action(ArgAction::Append)
                 .num_args(0..=2)
                 .value_hint(clap::ValueHint::FilePath),
-        )
+        );
+
+    for i in ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"] {
+        //can't iterate over {0..=9} because it doesn't support Into<Id>
+        res = res.arg(Arg::new(i).short(i.chars().next().unwrap()).num_args(0..=1));
```

Comment:
These all need to be marked hidden still. It currently shows up like this in the help, which is not great:
```
Options:
  -D, --all-repeated[=<delimit-method>]  print all duplicate lines. Delimiting is done with blank lines. [default: none] [possible values: none, prepend, separate]
      --group[=<group-method>]           show all items, separating groups with an empty line. [default: separate] [possible values: separate, prepend, append, both]
  -w, --check-chars <N>                  compare no more than N characters in lines
  -c, --count                            prefix lines by the number of occurrences
  -i, --ignore-case                      ignore differences in case when comparing
  -d, --repeated                         only print duplicate lines
  -s, --skip-chars <N>                   avoid comparing the first N characters
  -f, --skip-fields <N>                  avoid comparing the first N fields
  -u, --unique                           only print unique lines
  -z, --zero-terminated                  end lines with 0 byte, not newline
  -0 [<0>]                               
  -1 [<1>]                               
  -2 [<2>]                               
  -3 [<3>]                               
  -4 [<4>]                               
  -5 [<5>]                               
  -6 [<6>]                               
  -7 [<7>]                               
  -8 [<8>]                               
  -9 [<9>]                               
  -h, --help                             Print help information
  -V, --version                          Print version information
```